app/main.py

```diff
@@ -1 +1,11 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command = command.split()
+    if command[0] == "cp" and command[1] != command[2]:
+        with open(command[1], "r") as file_input:
+            with open(command[2], "w") as file_output:
```

Comment:
You can nest the context managers using a single with statement and separating them with commas to achieve a one-liner for opening both files